docs/faq.md

```diff
@@ -130,21 +130,23 @@ Run "docker system df" to see how much space can be reclaimed. If the space is l
 
 Run command "du -h / | awk '$1~/[0-9]*G/{print $0}'" to list all the directory which consumes more than 1G spaces. Clean the data not used any more. 
 
-### Q: How to change Docker install path for PAI?
+### Q: How to change Docker cache path for OpenPAI?
 
-A: If we install Docker on a small disk, when under disk pressure, the cleaner may activate and kill jobs to prevent system crash. We'd better to move Docker to a larger disk. We can change Docker install path through following steps:
+A: If default Docker cache path is set to a disk with small disk space, the path can be changed with below steps:
 
-1.	Change the Docker path on PAI node.  
-2.	Stop ALL service of the cluster.  
-```sudo ./paictl.py service stop```
-3.	Destroy k8s cluster.  
+1.	Stop ALL service of the cluster.  
+  ```sudo ./paictl.py service stop```
+2.	Destroy k8s cluster.  
```

Comment:
I prefer separate step 3 and 4 into two steps. Because one is for Docker and the other one is for OpenPAI.
I can mention the docker path in step 4 is the configured path in step 3.

Change destroy to uninstall

Reply:
I mean create an issue to simplify it, so that user can update configuration easier. And we have a chance to clean up old folder, if it's changed.